spec/quickstart_spec.rb

```diff
@@ -3,8 +3,9 @@
 describe "quickstart" do
     it "generates percl to say the text 'Hello, World!'" do
         say = Freeclimb::Say.new(text:'Hello, World!')
-        script = Freeclimb::PerclScript.new(commands:[say])
+        getSpeech = Freeclimb::GetSpeech.new(action_url: 'https://example.com/update')
```

Comment:
I think we may need to snake case this variable since ruby follows snake case format